newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -261,20 +261,30 @@ export const HomePage = React.memo<Props>(
       const createBlankProject = async (i18n: I18nType) => {
         setIsOpening(true);
         try {
+          // Create project.
           const projectMetadata = await onCreateBlank({
             i18n,
             outputPath,
             projectName: newProjectName,
           });
           if (!projectMetadata) return;
-          const { project, storageProvider, fileMetadata } = projectMetadata;
+          const {
+            project,
+            storageProvider,
+            fileMetadata,
+            projectName,
+          } = projectMetadata;
+
+          // Once project is created, reinitialize default values for variables related to project creation.
```

Comment:
Thanks for the comment. Any chance this could be moved at the moment the project is created, rather than having to remember to "clean" after the creation?
Not sure if it makes sense, but I figure it's more intuitive to have set at the right moment before creating/opening a dialog than having to remember to clean these later.